### PR TITLE
Changed Structure

### PR DESCRIPTION
**Pull request by jeppard**
# Skiosa - PR
- [x] documentation can't be split up into smaller logical units

### CODE REVIEW

**Comment by grossamos** [comment]
Could you possibly comment on what should go into `bearbeitete user stories`, should we document all user stories, or just "the important ones"?

I think It's not 100% nessicary to document every story we have.

**Comment by grossamos** [comment]
Additionally I only see a spot for the initial architecture.

Since a lot has changed and we need our sequence diagrams (SKIOS-125) to also find a place, this would be great if clarified.

**Comment by grossamos** [comment]
Also unclear is where organisational changes (such as our estimations, etc.) that happened over time will be, is it a part of team organisation or (because they are the result of retros) a part of our sprints?

**Comment by grossamos** [comment]
The location of our Mockups is also not 100% clear, Garidis mentioned that we should list and explain them:

> Mock-ups (or as he calls them paper prototypes) should be listed and explained in the order that they were created, explaining the process behind how we designed our platform. (Design language, colors, components, frameworks, pages, etc.)

 (see https://skiosa.atlassian.net/l/c/03rLKGBW)

I don't quite see this requirement fulfilled in a story by story explanation of how we created mockups (in `bearbeitete user stories`).

**Comment by grossamos** [comment]
Also (and this is just an idea), we could possibly add a summary section containing a table of organisational, archtectural and code changes to kinda sum up our work

**Comment by jeppard** [comment]
> Could you possibly comment on what should go into `bearbeitete user stories`, should we document all user stories, or just "the important ones"?
> 
> I think It's not 100% nessicary to document every story we have.

We should document all stories, because there is not a thing as unimportant stories. If they would be unimportant they would not exist. (Apart from the wont-fix stories)

**Comment by jeppard** [comment]
> 

Architecture should be part of code documentation in the appendix

**Comment by jeppard** [comment]
> 

Inside sprints, they were discussed there

**Comment by jeppard** [comment]
> 

Mockups are a result of a story so inside the story

**Comment by jeppard** [comment]
> The location of our Mockups is also not 100% clear, Garidis mentioned that we should list and explain them:
> 
> > Mock-ups (or as he calls them paper prototypes) should be listed and explained in the order that they were created, explaining the process behind how we designed our platform. (Design language, colors, components, frameworks, pages, etc.)
> 
> (see https://skiosa.atlassian.net/l/c/03rLKGBW)
> 
> I don't quite see this requirement fulfilled in a story by story explanation of how we created mockups (in `bearbeitete user stories`).

Do you have a solution in mind?

**Comment by grossamos** [comment]
> Architecture should be part of code documentation in the appendix

Hm, as a PO, I would like to put stories such as SKIOS-122 and SKIOS-125 outside of the appendix. 
Goal of the documentation (or SWE as a whole) is to showcase how we can use project management methods to tackle tasks in software development. Since these stories showcase exactly that, let's find a place for architecture **outside** of the Appendix.

File: content/05kapitel.tex [hunk 1]
```diff
@@ -0,0 +1,10 @@
+% !TeX root = ../dokumentation.tex
+
+\chapter{Projekt Ergebnisse}
```
Comment (grossamos) [inline content/05kapitel.tex:3]: regarding our organisational changes (which you clarified should be mentioned in our section for sprints), could we add a section here elaborating on analysis of their results?

**Comment by grossamos** [comment]
> Do you have a solution in mind?

How about we add a section for our "business"-related work (dt. Fachlichkeit, also die Inhaltlichen Themen unseres Projektes), containing:
- Mockups (design language, logo, colors, etc.)
- Architecture
- Changes regarding how we deal with RSS feeds (since the initial requirements will reflect our handling of RSS feeds differently than our current understanding)

I'd place it between chapter `Sprints` and `Projekt Ergebnisse` and call it something like `Fachliche Ausarbeitung des Projektes` or something like that (having trouble finding a good name)

**Comment by jeppard** [comment]
> SKIOS-122

Your searching for Section 1.2?

File: content/05kapitel.tex (hunk 1 above)
Comment (jeppard) [reply content/05kapitel.tex:3]: Good point omw to change

**Comment by jeppard** [comment]
> > Do you have a solution in mind?
> 
> How about we add a chapter for our "business"-related work (dt. Fachlichkeit, also die Inhaltlichen Themen unseres Projektes), containing:
> 
>     * Mockups (design language, logo, colors, etc.)
> 
>     * Architecture
> 
>     * Changes regarding how we deal with RSS feeds (since the initial requirements will reflect our handling of RSS feeds differently than our current understanding)
> 
> 
> I'd place it between chapter `Sprints` and `Projekt Ergebnisse` and call it something like `Fachliche Ausarbeitung des Projektes` or something like that (having trouble finding a good name)

I agree

**Comment by grossamos** [comment]
> > SKIOS-122
> 
> Your searching for Section 1.2?

our initial architecture includes things like the recomendation service and our **goal** for setting up the architecture (at least, that's what I'd expect from a chapter called `Projektziele`. SKIOS-122 includes an explanation for how this was then actually set up. 

(this would also be solved by just creating the chapter named above, so I'd be happy with that solution!)

**Comment by grossamos** [comment]
> > > Do you have a solution in mind?
> > 
> > 
> > How about we add a chapter for our "business"-related work (dt. Fachlichkeit, also die Inhaltlichen Themen unseres Projektes), containing:
> > ```
> > * Mockups (design language, logo, colors, etc.)
> > 
> > * Architecture
> > 
> > * Changes regarding how we deal with RSS feeds (since the initial requirements will reflect our handling of RSS feeds differently than our current understanding)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I'd place it between chapter `Sprints` and `Projekt Ergebnisse` and call it something like `Fachliche Ausarbeitung des Projektes` or something like that (having trouble finding a good name)
> 
> I agree

perfect, I'll approve as soon as that chapter is created :+1:

File: content/05kapitel.tex (hunk 1 above)
Comment (grossamos) [reply content/05kapitel.tex]: Seeing how you added a couple of commits, which section is this?

**Comment by jeppard** [comment]
> > > > Do you have a solution in mind?
> > > 
> > > 
> > > How about we add a chapter for our "business"-related work (dt. Fachlichkeit, also die Inhaltlichen Themen unseres Projektes), containing:
> > > ```
> > > * Mockups (design language, logo, colors, etc.)
> > > 
> > > * Architecture
> > > 
> > > * Changes regarding how we deal with RSS feeds (since the initial requirements will reflect our handling of RSS feeds differently than our current understanding)
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > I'd place it between chapter `Sprints` and `Projekt Ergebnisse` and call it something like `Fachliche Ausarbeitung des Projektes` or something like that (having trouble finding a good name)
> > 
> > 
> > I agree
> 
> perfect, I'll approve as soon as that chapter is created 👍

Changes done

**Comment by grossamos** [comment]
Perfect, only one point left (see thread above):
> Seeing how you added a couple of commits, which section is this?

(wasn't able to find the section where we're documenting and analysing organizational changes, was this possibly not added?)

**Comment by jeppard** [comment]
Yes missed IT because file Was mived will do tomorow

**Review by grossamos — APPROVED** [review]
looks good, can be merged

**Pull request merged**